client/client.go

```diff
@@ -619,13 +620,14 @@ func (c *NotaryClient) InitRepository(gun string, baseURL string, transport http
 		return nil, err
 	}
 
-	signer := signed.NewSigner(NewCryptoService(gun, *privKeyStore))
+	signer := signed.NewSigner(NewCryptoService(gun, privKeyStore))
 
 	nRepo := &NotaryRepository{Gun: gun,
 		baseURL:          baseURL,
 		tufRepoPath:      filepath.Join(c.baseDir, tufDir, gun),
 		transport:        transport,
 		signer:           signer,
+		privKeyStore:     privKeyStore,
```

Comment:
Correct. I think it's worth having here. I can add a comment saying only tests use it right now, if you want.